gen3users/user_security.py

```diff
@@ -107,58 +109,80 @@ def _check_yaml(
 
         logging.info(f"========= Scanning file for {phsid} in open projects =========")
         found = False
-        # Track down open_data_reader
+
+        # Track down open data policies (anonymous or all_users)
         policies = content[auth_meth]["policies"]
```

Comment:
It's probably worth wrapping this in a try block also, just in case the format isn't as expected.